static/src/javascripts/lib/page.js

```diff
@@ -4,9 +4,13 @@ import $ from 'lib/$';
 import config from 'lib/config';
 import { getBreakpoint } from 'lib/detect';
 
-type yesable = ?(arg: boolean | string) => boolean | void;
+// #? this should'nt really be an `any`, but the callbacks themselves are explicit
+// about the types they accept.
+// that this is required suggests maybe the approach is too generic
+type yesable = ?(arg: any) => boolean | void;
```

Comment:
Can we go with [`mixed`](https://flow.org/en/docs/types/mixed/#anything-goes-in-nothing-comes-out-a-classtoc-idtoc-anything-goes-in-nothing-comes-out-hreftoc-anything-goes-in-nothing-comes-outa) rather than `any`? This might incur further refinement

Reply:
Can we do this in a seperate PR?

Reply:
Yep. It has a comment, so we'll find it. Also I want to kill all the `any`s, so this will happen as long as I still draw breath